src/WebJobs.Extensions.DurableTask/DurableTaskJobHostConfigurationExtensions.cs

```diff
@@ -41,7 +41,7 @@ public static IServiceCollection AddDurableClientFactory(this IServiceCollection
             }
 
             serviceCollection.TryAddSingleton<INameResolver, DefaultNameResolver>();
-            serviceCollection.TryAddSingleton<IConnectionStringResolver, StandardConnectionStringProvider>();
+            serviceCollection.TryAddSingleton<IConnectionStringResolver, WebJobsConnectionStringProvider>();
```

Comment:
I am a little concerned about changing this, just because some customers will be running in non-webjobs scenario.

Reply:
I initially changed it because of this comment in the [linked issue](https://github.com/Azure/azure-functions-durable-extension/issues/1656#issue-790126163):

> In addition when consuming this service via non web job (standard aspnet application) the extention method defaults to using the implementation of IConnectionStringProvider as StandardConnectionStringProvider.
> 
> After looking the source code it appears that the standard implementation would be better suited using WebJobsConnectionStringProvider this will locate connection string via both local.settings.json:Values:ConnString in webjob and appsettings.json:ConnectionStrings:ConnString in standard ASP.NET application and therefore feels like the naming convention of these implementations are reversed.

I can change it back to `StandardConnectionStringProvider`. I looked through `StandardConnectionStringProvider` and `WebJobsConnectionStringProvider` to understand this comment and the only difference I see is that `WebJobsConnectionStringProvider` evaluates `configuration.GetConnectionString(connectionName)` first before `configuration[connectionName]`. I'm not exactly clear on what `configuration.GetConnectionString(connectionName)` does/where it looks for connection strings. Let me know if you think `WebJobsConnectionStringProvider` is fine otherwise I'll change it to `StandardConnectionStringProvider`

https://github.com/Azure/azure-functions-durable-extension/blob/cc7f8b20844a07b0729a3ae66bcc5dcf156baaf3/src/WebJobs.Extensions.DurableTask/StandardConnectionStringProvider.cs#L31

https://github.com/Azure/azure-functions-durable-extension/blob/cc7f8b20844a07b0729a3ae66bcc5dcf156baaf3/src/WebJobs.Extensions.DurableTask/WebJobsConnectionStringProvider.cs#L29

https://github.com/Azure/azure-webjobs-sdk/blob/4a16c273b40ee47ec318bb6d7aa12e0075f30402/src/Microsoft.Azure.WebJobs.Host/Extensions/IConfigurationExtensions.cs#L79